array_test.go

```diff
@@ -952,3 +953,114 @@ func TestArrayFilter(t *testing.T) {
 			filteredArray.chain.assertNotFailed(t)
 		})
 }
+
+func TestArray_IsOrdered(t *testing.T) {
```

Comment:
Nitpick: we don't use underscores in test names in this project.

Reply:
Actually using underscores is a good idea, I refactored tests: cea445a6313289ca84f6f33c83ab4013bc59adc2

So this is not relevant, but please rebase on fresh master.

Reply:
thanks, FYI I generate test skeleton from VS Code, it uses underscore for the naming